omnigibson/examples/object_states/particle_source_sink_demo.py

```diff
@@ -51,16 +51,16 @@ def check_toggledon(obj):
                 "conditions": {
                     "water": [check_toggledon],   # Must be toggled on for water source to be active
                 },
-                "source_radius": 0.025,
-                "source_height": 0.10,
+                "source_radius": 0.0125,
+                "source_height": 0.05,
                 "initial_speed": 0.0,               # Water merely falls out of the spout
             },
             "particleSink": {
                 "conditions": {
                     "water": None,  # No conditions, always sinking nearby particles
                 },
-                "sink_radius": 0.20,
-                "sink_height": 0.20,
+                "sink_radius": 0.05,
```

Comment:
In this case it's 4x smaller -- was this just empirically tuned?

Reply:
Yeah I think it's more realistic that way. Otherwise the drainage hole is going to be really large.